openshift-container-platform-4/policies/SI-System_and_Information_Integrity/component.yaml

```diff
@@ -89,6 +89,25 @@
         benchmarks to remediate high, and moderate risk flaws within a
         customer defined period after a flaws discovery.
         */'
+
+- control_key: SI-2(4)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: complete
+  narrative:
+    - key: a
+      text: |
```

Comment:
```suggestion
```

Reply:
Wish GitHub had a multi-line suggestion tool!

Because this control does not have any sub elements, such as how SI-2 has part A, B C, and D, there is no need to further enumerate the subcontrols.